scripts/update-version-module.js

```diff
@@ -31,15 +68,19 @@ import {fileURLToPath} from 'url';
 
 // This file is automatically generated by scripts/update-version-module.js
 // before publishing.
-export const version = '${version}';
+export const npmVersion = '${await getNpmVersion()}';
+export const serviceWorkerHash = '${await getServiceWorkerHash()}';
 `;
 
-  const versionModulePath = join(thisDir, '..', 'src', 'shared', 'version.ts');
-  const oldVersionModuleCode = await fs.readFile(versionModulePath, 'utf8');
-  if (oldVersionModuleCode !== newVersionModuleCode) {
-    await fs.writeFile(versionModulePath, newVersionModuleCode, 'utf8');
-    console.log('Updated src/shared/version.ts, please commit this change.');
-    // Fail to help the publisher remember to commit.
-    process.exit(1);
-  }
-})();
+const versionModulePath = join(thisDir, '..', 'src', 'shared', 'version.ts');
+const oldVersionModuleCode = await fs.readFile(versionModulePath, 'utf8');
```

Comment:
fwiw, there's three i/o operations here that could be parallelized here if there were lifted up to a `Promise.all()`

Reply:
Yeah, but this runs so rarely and is so fast that I'd rather just keep the code as simple as possible.